vms/proposervm/block.go

```diff
@@ -199,6 +204,11 @@ func (p *postForkCommonComponents) buildChild(
 		proposerID := p.vm.ctx.NodeID
 		minDelay, err := p.vm.Windower.Delay(ctx, parentHeight+1, parentPChainHeight, proposerID)
 		if err != nil {
+			p.vm.ctx.Log.Error("unexpected build block failure",
+				zap.String("reason", "failed to calculate required timestamp delay"),
+				zap.Stringer("parentID", parentID),
+				zap.Error(err),
+			)
 			return nil, err
 		}
 
```

Comment:
Notice that we do not log an error below here if `if delay < minDelay {` as that can happen normally and is handled gracefully.